### DISCUSSION TITLE
Problem creating Tar  > 10 Go

### DISCUSSION THREAD

**Discussion by delayes** [icsharpcode/SharpZipLib#858] [Q&A]
Hello,  

I have a question concerning the creation of large  TAR , I noticed I have an issue when they are above 8 Go

When  I check the tar I created with  7zip    I can see that   the caracteristics have the following.
![image](https://github.com/icsharpcode/SharpZipLib/assets/6288877/8be866c7-a048-4914-a927-caaf4175e185)

That's an issue because when I check the header of  my tar I have an exception  "Could not find any recognizable digits."

I would like to know if someone else have the same issue or if I don't   compress the good way.

.net core 3.1 

```
         using (var tarFileStream = new FileStream(destinationTarFile, FileMode.Create))
            using (var tarOutputStream = new TarOutputStream(tarFileStream,))
            {
                CompressDirectory(sourceDirectory, tarOutputStream);
            }


 static void CompressDirectory(string sourceDirectory, TarOutputStream tarOutputStream)
 {
     foreach (var filePath in Directory.EnumerateFiles(sourceDirectory, "test.xx", SearchOption.AllDirectories))
     {
         var relativePath = Path.GetRelativePath(sourceDirectory, filePath);
         var entry = TarEntry.CreateTarEntry(relativePath);

         using (var fileStream = new FileStream(filePath, FileMode.Open, FileAccess.Read))
         {
             entry.Size = fileStream.Length;
             tarOutputStream.PutNextEntry(entry);        
             fileStream.CopyTo(tarOutputStream);
         }
     }
 }
```

**Comment by SourceproStudio** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
